packages/react/src/context/GlobalStyles.tsx

```diff
@@ -0,0 +1,35 @@
+import { Global, css } from '@emotion/react';
+
+const globalStyles = css`
+  body {
+    font-family: 'Arial, sans-serif';
+    margin: 0;
+    padding: 0;
+    color: #484848;
+  }
+
+  * {
+    box-sizing: border-box;
+  }
+
+  h1 {
```

Comment:
Headings are not global styles. That must be done inside theme object. Here is a list of good resets to add to global styles: https://github.com/Lazzzer00/Best-CSS-Reset-2024